pywal/export.py

```diff
@@ -12,8 +13,31 @@ def template(colors, input_file, output_file=None):
     """Read template file, substitute markers and
        save the file elsewhere."""
     template_data = util.read_file_raw(input_file)
-    template_data = "".join(template_data).format(**colors)
 
+    # Match all substitution markers
+    regex = r"(?<!{){[^{|}]+}"
```

Comment:
This can be moved to the next line (so it only takes up one line). `regex` is only used once.

Reply:
Ok, I've fixed this, and will upload soon.